packages/java/endpoint/src/main/java/dev/hilla/crud/PropertyStringFilterSpecification.java

```diff
@@ -0,0 +1,64 @@
+package dev.hilla.crud;
+
+import jakarta.persistence.criteria.CriteriaBuilder;
+import jakarta.persistence.criteria.CriteriaQuery;
+import jakarta.persistence.criteria.Expression;
+import jakarta.persistence.criteria.Path;
+import jakarta.persistence.criteria.Predicate;
+import jakarta.persistence.criteria.Root;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.TemporalAccessor;
+
+import dev.hilla.crud.filter.PropertyStringFilter;
+import org.springframework.data.jpa.domain.Specification;
+
+public class PropertyStringFilterSpecification<T> implements Specification<T> {
+
+    private PropertyStringFilter filter;
+    private Class<T> entity;
+    private Class<?> javaType;
+
+    public PropertyStringFilterSpecification(PropertyStringFilter filter,
+            Class<T> entity, Class<?> javaType) {
+        this.filter = filter;
+        this.entity = entity;
+        this.javaType = javaType;
+    }
+
+    @Override
+    public Predicate toPredicate(Root<T> root, CriteriaQuery<?> query,
+            CriteriaBuilder criteriaBuilder) {
+        String value = filter.getFilterValue();
+        Path<String> propertyPath = root.get(filter.getPropertyId());
```

Comment:
Looks like this will throw should be path / property not exist. Intended, or should we continue and log a warning?

Reply:
added a test